cola/themes.py

```diff
@@ -12,6 +13,7 @@
 class EStylesheet(object):
     DEFAULT = 1
     FLAT = 2
+    CUSTOM = 3
```

Comment:
Nicely done. Yup, this is pretty much exactly what I had in mind on how to implement this feature.